src/corehost/packaging/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -0,0 +1,47 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <PropertyGroup>
+    <Version>1.0.0</Version>
+  </PropertyGroup>
+
+  <Target Name="ReadDependenciesFromProjectJsonFile" BeforeTargets="GetPackageDependencies">
+    <ReadLinesFromFile File="$(MSBuildThisFileDirectory)project.json">
+      <Output TaskParameter="Lines" ItemName="ProjectJsonLines" />
+    </ReadLinesFromFile>
+
+    <ItemGroup>
+      <PackageMatch Include="@(ProjectJsonLines)">
+        <PackageId>$([System.Text.RegularExpressions.Regex]::Match('%(Identity)', '(\w[^:&quot;]*)'))</PackageId>
+        <PackageVersion>$([System.Text.RegularExpressions.Regex]::Match('%(Identity)', '(\d+\.\d+.\d+[^&quot;]*)'))</PackageVersion>
+      </PackageMatch>
+
+      <Dependency Include="@(PackageMatch -> '%(PackageId)')" Condition="'%(PackageVersion)' != ''">
+        <Version>%(PackageVersion)</Version>
+        <TargetFramework>.NETStandard1.0</TargetFramework>
+      </Dependency>
+
+      <!-- List of dependencies to mark as exclude=compile -->
+      <Dependency Condition="'%(Dependency.Identity)' == 'Microsoft.CodeAnalysis.CSharp'">
+        <Exclude>Compile</Exclude>
+      </Dependency>
+      <Dependency Condition="'%(Dependency.Identity)' == 'Microsoft.CodeAnalysis.VisualBasic'">
+        <Exclude>Compile</Exclude>
+      </Dependency>
+      <Dependency Condition="'%(Dependency.Identity)' == 'System.Runtime.Loader'">
+        <Exclude>Compile</Exclude>
+      </Dependency>
+    </ItemGroup>
+
+    <!-- Use NETStandard.Library version suffix for version suffix of Microsoft.NETCore.App -->
+    <PropertyGroup>
```

Comment:
Oops I just responded to your other question pointing at this place. I can update the comment.